wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/ConversationListViewController+NavigationBar.swift

```diff
@@ -213,9 +214,34 @@ extension ConversationListViewController {
         }
 
         action.setValue(attributedTitle, forKey: "attributedTitle")
+        action.accessibilityLabel = setAccessibilityLabelForFilterAction(for: filter, isSelected: isSelected)
+
         return action
     }
 
+    func setAccessibilityLabelForFilterAction(
+        for filter: ConversationFilterType?,
+        isSelected: Bool
+    ) -> String {
+
+        typealias FilterMenuOptionsAccessibilityLocale = L10n.Accessibility.ConversationsList.FilterMenuOptions
```

Comment:
```suggestion
        typealias accessibilityLocale = L10n.Accessibility.ConversationsList.FilterMenuOptions
```

Reply:
to shorten a bit the variable

Reply:
Done: https://github.com/wireapp/wire-ios/commit/9df53047654263fea3bad39f6d964fffa4d3ea76